### PR TITLE
Unit tests for `parse_json_markdown` function

### PR DESCRIPTION
**Pull request by tisnik**
## Type of change

- [x] Unit tests
- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Unit tests for `parse_json_markdown` function. Checks correct inputs and also some incorrect ones.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- It is part of unit testing job

### CODE REVIEW

File: tests/unit/utils/test_json_tools.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+)
+
+
+@pytest.mark.parametrize("json,expected", correct_inputs)
+def test_parse_json_markdown(json, expected):
+    """Check parsing correct input."""
+    assert parse_json_markdown(json) == expected
+
+
+incorrect_inputs = (None, "", "foo", "\n", '"')
```
Comment (bparees) [inline tests/unit/utils/test_json_tools.py:23]: how about a test in which the content inside the markdown/backticks is not json?

Comment (tisnik) [reply tests/unit/utils/test_json_tools.py]: yup, added

**Pull request merged**